src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py

```diff
@@ -147,6 +147,15 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
                    action='store_true',
                    arg_group='Basic Policy Options',
                    help='Use open restriction. License or key will be delivered on every request.')
+        c.argument('policy_option_name',
+                   help='The name of the policy option.')
+
+    with self.argument_context('ams content-key-policy') as c:
+        c.argument('with_secrets',
+                   action='store_true',
+                   help='Include secret values of the content key policy.')
+
+    with self.argument_context('ams content-key-policy options add') as c:
```

Comment:
When adding all this, ams content-key-policy create would not know the rest of the flags, hence not showing description for them. They should be on a higher group IMO.